style.css

```diff
@@ -0,0 +1,32 @@
+body {
+  background-color: rgb(56, 48, 163);
+}
+
+#survey-form {
+  width: 480px;
+  margin: auto;
+  position: middle;
```

Comment:
- [ ] Good! Kindly take note that the line of code may not be invalid as stated; but perhaps according to the template design it may not be needful. Please you need to address the developer to ensure that the design is close to the given template. Smile